### PR TITLE
BaSD-MS-22-Sessions:  Create React Logic

### PR DESCRIPTION
**Pull request by ignacioaristo**
Resume:
Frontend was connected with backend using React.

Methods developed for Sessions:

Get all the Sessions

Create a Session

Click on Add Session button to display the form.

Update a Sessions
Click on the row of the Session to display the form and rewrite it.

Delete a Session
Click on the trash can icon to delete the Session.

Modal was created for delete option.

Validations were added to every form.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
+      "no-unused-vars": "warn",
+      "semi": [2, "always"],
+      "react/react-in-jsx-scope": "off",
+      // "prettier/prettier": [
+      //   "error",
+      //   {
+      //     "endOfLine": "auto"
+      //   }
+      // ],
+      "react/prop-types": 0
```
Comment (aquilesmariani) [inline .eslintrc.json:31]: Let's get rid of these changes

**Review by aquilesmariani — CHANGES_REQUESTED** [review]
Please check on the comments on the [clients' PR](https://github.com/BaSD-2021/Aquiles-MindSet-app/pull/1#pullrequestreview-820985070) and apply them here as well since most are repeated.

File: .eslintrc.json (hunk 1 above)
Comment (ignacioaristo) [reply .eslintrc.json]: Chages removed

**Review by aquilesmariani — APPROVED** [review]
LGTM!

**Pull request merged**